test/federated/test_train_config.py

```diff
@@ -351,6 +352,84 @@ def forward(self, x):
     assert loss_after < loss_before
 
 
+@pytest.mark.asyncio
+async def test_train_config_with_jit_trace_async_nc(hook, start_proc):  # pragma: no cover
+    kwargs = {"id": "async_fit", "address": "ws://localhost:8777", "hook": hook}
```

Comment:
Amazing, how does this works? Does start_proc give you everything you need to build a NodeClient?

Reply:
Actually the testing code is copied from `test_train_config_with_jit_trace_async_wc` because i'm not familiar with the testing code....
It seems that `start_proc` is not called in this function, everything I need to build a NodeClient is from kwargs.